### PR TITLE
Manual installation jenkins

### PR DESCRIPTION
**Pull request by drodriguezhdez**

### CODE REVIEW

File: docs/pipelines/jenkins.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ To enable the APM traces collection for Jenkins builds and pipelines:
 
 APM Traces feature is only available if the `Datadog Agent` mode is selected.
 
+### Manual installation from your local station
```
Comment (soyguijarro) [inline docs/pipelines/jenkins.md:20]:
```suggestion
### Manual installation from your local machine
```

File: docs/pipelines/jenkins.md [hunk 2]
```diff
@@ -17,6 +17,14 @@ To enable the APM traces collection for Jenkins builds and pipelines:
...
+During the alpha phase, you could need install a release candidate artifact from your local station manually.
```
Comment (soyguijarro) [inline docs/pipelines/jenkins.md:21]:
```suggestion
During the alpha phase, you might need to install a release candidate artifact from your local machine manually.
```

File: docs/pipelines/jenkins.md [hunk 3]
```diff
@@ -17,6 +17,14 @@ To enable the APM traces collection for Jenkins builds and pipelines:
...
+
+To do that, you can follow the next steps:
+* Access to `$JENKINS_URL/pluginManager/advanced`
+* In the `Upload Plugin` section, upload the `datadog.hpi` artifact from your local station.
```
Comment (soyguijarro) [inline docs/pipelines/jenkins.md:25]:
```suggestion
* In the `Upload Plugin` section, upload the `datadog.hpi` artifact from your local machine.
```

File: docs/pipelines/jenkins.md (hunk 1 above)
Comment (alxch-) [inline docs/pipelines/jenkins.md:20]: is it going to override the existing version of the plugin? or do they need to delete anything first?

**Review by soyguijarro — COMMENTED** [review]
Some writing suggestions :)

File: docs/pipelines/jenkins.md (hunk 1 above)
Comment (drodriguezhdez) [reply docs/pipelines/jenkins.md]: This will override the existing version of the plugin. Adding it to the docs.

Comment (drodriguezhdez) [reply docs/pipelines/jenkins.md]: Added 👍🏼

**Review by alxch- — APPROVED** [review]

**Pull request merged**